utils/coreutils/techutils.go

```diff
@@ -164,3 +170,16 @@ func ToTechnologies(args []string) (technologies []Technology) {
 	}
 	return
 }
+
+func GetTechnologyDependencyFile(tech string) string {
+	techData, ok := technologiesData[Technology(tech)]
+	var dependencyFile string
+	if ok {
+		dependencyFile = techData.dependencyFile
+	}
+	if dependencyFile == "" {
+		return tech + "Dependency File"
```

Comment:
What happens in this case? Please explain

Reply:
In case we didn't find a matching package descriptor to the technology that was supplied in the parameter, as 'Default' string will be displayed. Something like 'NuGet Package Descriptor" (Changed to package descriptor according your comments)